src/allmydata/storage_client.py

```diff
@@ -1218,6 +1232,23 @@ def done(_):
         self._connecting_deferred = connecting
         return connecting
 
+    async def _agent_factory(self) -> Optional[Callable[[object, IPolicyForHTTPS, HTTPConnectionPool],IAgent]]:
+        """Return a factory for ``twisted.web.iweb.IAgent``."""
+        # TODO default_connection_handlers should really be an object, not a dict...
```

Comment:
Might be better to file a ticket and link?

(It's not clear to me _why_ it should be an object, or what API it might have, although I can make some guesses ;)